apps/src/util/experiments.js

```diff
@@ -82,12 +89,19 @@ experiments.setEnabled = function (key, shouldEnable, expiration=undefined) {
  * @returns {bool}
  */
 experiments.isEnabled = function (key) {
-  let enabled = this.getStoredExperiments_()
+  const storedExperiments = this.getStoredExperiments_();
+  let enabled = storedExperiments
     .some(experiment => experiment.key === key) ||
     !!(window.appOptions &&
       window.appOptions.experiments &&
       window.appOptions.experiments.includes(key));
```

Comment:
Is there a scenario where an experiment in `window.appOptions.experiments` would not already have been returned to us by the `experimentsCookie` logic in `getStoredExperiments_` above? If not, we could consider removing the `window.appOptions` logic in a future PR.

Reply:
Good idea - I don't think so.